go/signedexchange/signedexchange.go

```diff
@@ -504,3 +507,40 @@ func (e *Exchange) PrettyPrintPayload(w io.Writer) {
 	fmt.Fprintf(w, "payload [%d bytes]:\n", len(e.Payload))
 	w.Write(e.Payload)
 }
+
+func (e *Exchange) JSONPrint(w io.Writer) {
+	verificationTime := time.Now()
```

Comment:
Would you pass verificationTime via this func's argument?

Reply:
Done.